cmd/system-probe/main.go

```diff
@@ -54,108 +30,29 @@ func main() {
 	}
 	defer log.Flush()
 
-	// --version
-	if opts.version {
-		fmt.Println(versionString("\n"))
-		os.Exit(0)
-	}
-
-	// --pid
-	if opts.pidFilePath != "" {
-		if err := pidfile.WritePID(opts.pidFilePath); err != nil {
-			log.Errorf("Error while writing PID file, exiting: %v", err)
-			os.Exit(1)
-		}
-
-		log.Infof("pid '%d' written to pid file '%s'", os.Getpid(), opts.pidFilePath)
-
-		defer func() {
-			os.Remove(opts.pidFilePath)
-		}()
-	}
-
-	// Parsing YAML config files
-	cfg, err := config.NewSystemProbeConfig(loggerName, opts.configPath)
-	if err != nil {
-		log.Criticalf("Failed to create agent config: %s", err)
-		os.Exit(1)
-	}
+	exit := make(chan bool)
```

Comment:
This isn't being used atm. `runAgent()` is recreating this channel 	

```
e := make(chan bool)
go util.HandleSignals(e)
<-e
```

You can remove it as a parameter